index.html

```diff
@@ -1,71 +1,12 @@
 <!DOCTYPE html>
 <html lang="kr">
-  <head>
-    <meta charset="UTF-8" />
-    <title>🏎️ 자동차 경주 게임</title>
-    <link rel="stylesheet" href="./src/css/index.css" />
-  </head>
-  <body>
-    <div id="app">
-      <section class="d-flex justify-center mt-5">
-        <form>
-          <fieldset>
-            <h1 class="text-center">🏎️ 자동차 경주 게임</h1>
-            <p>
-              5자 이하의 자동차 이름을 콤마로 구분하여 입력해주세요. <br />
-              예시) EAST, WEST, SOUTH, NORTH
-            </p>
-            <div class="d-flex">
-              <input type="text" class="w-100 mr-2" placeholder="자동차 이름" />
-              <button type="button" class="btn btn-cyan">확인</button>
-            </div>
-          </fieldset>
-          <fieldset>
-            <p>시도할 횟수를 입력해주세요.</p>
-            <div class="d-flex">
-              <input type="number" class="w-100 mr-2" placeholder="시도 횟수" />
-              <button type="button" class="btn btn-cyan">확인</button>
-            </div>
-          </fieldset>
-        </form>
-      </section>
-      <section class="d-flex justify-center mt-5">
-        <div class="mt-4 d-flex">
-          <div class="mr-2">
-            <div class="car-player">EAST</div>
-            <div class="forward-icon mt-2">⬇️️</div>
-            <div class="forward-icon mt-2">⬇️️</div>
-          </div>
-          <div class="mr-2">
-            <div class="car-player">WEST</div>
-            <div class="forward-icon mt-2">⬇️️</div>
-          </div>
-          <div class="mr-2">
-            <div class="car-player">SOUTH</div>
-            <div class="d-flex justify-center mt-3">
-              <div class="relative spinner-container">
-                <span class="material spinner"></span>
-              </div>
-            </div>
-          </div>
-          <div class="mr-2">
-            <div class="car-player">NORTH</div>
-            <div class="d-flex justify-center mt-3">
-              <div class="relative spinner-container">
-                <span class="material spinner"></span>
-              </div>
-            </div>
-          </div>
-        </div>
-      </section>
-      <section class="d-flex justify-center mt-5">
-        <div>
-          <h2>🏆 최종 우승자: EAST, WEST 🏆</h2>
-          <div class="d-flex justify-center">
-            <button type="button" class="btn btn-cyan">다시 시작하기</button>
-          </div>
-        </div>
-      </section>
-    </div>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<title>🏎️ 자동차 경주 게임</title>
+		<link rel="stylesheet" href="./src/css/index.css" />
+		<script type="module" src="./src/js/index.js" defer></script>
```

Comment:
`type="module"`일때는 자동으로 `defer` 속성이 적용된다고 하네요~
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts
```suggestion
		<script type="module" src="./src/js/index.js"></script>
```

Reply:
오 감사합니다~